src/prpy/base/manipulator.py

```diff
@@ -141,19 +141,25 @@ def SetVelocityLimits(self, velocity_limits, min_accel_time):
         self.GetRobot().SetDOFAccelerationLimits(or_accel_limits)
 
     def _PlanWrapper(self, planning_method, args, kw_args):
-        from prpy.clone import Clone, Cloned
+        from prpy.clone import Clone
         robot = self.GetRobot()
-        with Clone(robot.GetEnv()):
-            Cloned(self).SetActive()
+        with Clone(robot.GetEnv()) as cloned_env:
+            cloned_env.Cloned(self).SetActive()
```

Comment:
What was wrong with the previous `Cloned(self).SetActive()`. There are no threads here, so it should be unambiguous. Is this just a matter of preference?